arch/arc/Kconfig

```diff
@@ -406,4 +406,7 @@ config CMSIS_V2_THREAD_MAX_STACK_SIZE
 config CMSIS_V2_THREAD_DYNAMIC_STACK_SIZE
 	default 2048 if 64BIT
 
+config ARCH_HAS_DIRECTED_IPIS
+	default y
```

Comment:
Doing this with a select rom the ARCH_whatever kconfig is a little cleaner IMHO.

Reply:
I will see what I can do.